test/functional/test-video-manager.js

```diff
@@ -60,11 +62,203 @@ describes.fakeWin('VideoManager', {
     expect(spy).to.have.been.calledWith('unmute');
   });
 
+  it('should be paused if autoplay is not set', () => {
+    const klass = createFakeVideoPlayerClass(env.win);
```

Comment:
You can refactor these three lines in `beforeEach` (define variables outside of beforeEach and set them in here)